aten/src/ATen/native/quantized/cpu/qlinear_prepack.cpp

```diff
@@ -281,6 +282,25 @@ c10::intrusive_ptr<LinearPackedParamsBase> PackedLinearWeightsOnednn::prepack(
         bias});
   return ret_ptr;
 }
+
+inline at::Tensor pack_weight_to_onednn_tensor(
+    const at::Tensor& weight,
+    c10::optional<torch::List<int64_t>>& input_shape) {
+  std::vector<int64_t> w_dims = weight.sizes().vec();
+  ideep::tensor wei = ideep::tensor({w_dims, dnnl::memory::data_type::s8}, weight.data_ptr());
+  wei.transpose_(0, 1); // oneDNN requires transposed weight
+  ideep::dims input_dims = input_shape.has_value() ? input_shape.value().vec() : ideep::dims();
+  auto w_desc = ideep::matmul_forward::expected_weights_desc(
```

Comment:
Do we need to pass in the input zp and scale to query the weight format?

Reply:
Thanks. It's fixed.